sharktank/sharktank/models/grok/grok.py

```diff
@@ -123,7 +123,7 @@ def prefill(
         self._assert_device(seq_block_ids)
         self._assert_device(*cache_state, dtype=self.activation_dtype)
         h = self.token_embedding(tokens)
-        h *= math.sqrt(h.shape[-1])
+        h *= tanh.sqrt(h.shape[-1])
```

Comment:
You mean `torch.sqrt()`

Reply:
Also, everywhere we use `torch.sqrt()`, we need to make sure the input is in tensor form and not an `int`.

Reply:
changed back to `math.sqrt`. For any of the sqrt of dims we need to use the math variant. All of these are constants though so this should be fine.